source/index.js

```diff
@@ -32,10 +37,12 @@ var likely = function (node, options) {
  * Initiate Likely buttons on load
  */
 likely.initiate = likely.initate = function () {
-    var widgets = dom.findAll('.' + config.name);
+    var nodes = dom.findAll('.' + config.name);
```

Comment:
I’d better not do this. `widgets` is more specific and therefore more clear name.